docs/.vuepress/config.js

```diff
@@ -90,7 +106,8 @@ module.exports = {
         sidebar: convertSidebar(require('./sidebar/en.js'), '/en/')
       },
       '/zh-CN/': {
-        selectText: 'Languages',
+        algolia: buildAlgoliaSearchConfig('zh-CN'),
+        selectText: '选择语言',
```

Comment:
`选择语言` should always be in English. Both in `zh-CN` and `en` version.
For foreigner, they don't know and `选择语言` is `Language`. so that he can not found how to change language.